origin.spec

```diff
@@ -35,6 +35,9 @@
 %global make_redistributable 1
 %endif
 
+# by default build the test binaries for Origin
+%global build_tests 1
+
```

Comment:
If you did the following, then you could turn the build_tests off from the command line

`%{!?build_tests: %global build_tests 1 }`

And if you didn't want to build the tests, you would simply pass this on the command line (or via tito)

`rpmbuild -ba --define "build_tests 0" testrpm2-conditional.spec`